kasa/__init__.py

```diff
@@ -40,6 +40,7 @@
 from kasa.module import Module
 from kasa.protocols import BaseProtocol, IotProtocol, SmartProtocol
 from kasa.protocols.iotprotocol import _deprecated_TPLinkSmartHomeProtocol  # noqa: F401
+from kasa.smartcam.modules.camera import StreamResolution
```

Comment:
Do we want to lock this already to become the public API, or rather leave it until we have a stable camera interface?

Reply:
I think so because otherwise when HA starts using it it'll end up with another import from sub-packages and it's part of the method that uses it.